python_tests/test_python_frontend.py

```diff
@@ -539,6 +539,32 @@ def fusion_func(fd: FusionDefinition):
         eager_out = torch.sum(inputs[0].reshape(new_shape), dim=3)
         self.assertEqual(eager_out, nvf_out[0])
 
+    def test_execute_with_tuple_and_list(self):
+        shape = [2, 3, 4]
+        new_shape = [6, 4]
+
+        tensor = torch.randn(shape, device="cuda")
+        inputs_with_list = [tensor, new_shape]
+
+        def fusion_func(fd: FusionDefinition):
+            t0 = fd.from_pytorch(inputs_with_list[0])
+            n_shape = fd.define_vector(2)
+
+            t1 = fd.ops.reshape(t0, n_shape)
+            t2 = fd.ops.sum(t1, dims=[0])
+
+            fd.add_output(t2)
+
+        eager_out = torch.sum(inputs_with_list[0].reshape(new_shape), dim=0)
+
+        nvf_out, _ = self.exec_nvfuser(fusion_func, inputs_with_list)
+        self.assertEqual(eager_out, nvf_out[0])
+
+        inputs_with_tuple = [tensor, tuple(new_shape)]
```

Comment:
i realize it's not really related to this patch, but if the inputs themselves are tuples:
```python
inputs_with_tuple = (tensor, tuple(new_shape))
```
or even:
```python
inputs_with_tuple = (tensor, new_shape)
```
would we expect that to work? does it need a test?

Reply:
input here is a python iterable. so any container should work.

https://github.com/NVIDIA/Fuser/blob/af04aadb12b4b2e51eec5e17266f34e53b2a4430/csrc/python_frontend/python_bindings.cpp#L539

I wouldn't worry about it.